test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -561,17 +566,43 @@ scopes[upgrade_to_latest]="upgrade/downgrade databases to the latest versions of
 scopes[wt_standalone]="WiredTiger standalone releases"
 scopes[two_versions]="any two given versions"
 
-# The following associative array maps the 'official' build system to use for each branch.
-# CMake build support is reliably mature in newer release branches, whilst earlier revisions
-# primarily use Autoconf (note: some earlier branches may have CMake support, but these aren't
-# considered 'mature' versions.)
-declare -A build_sys
-build_sys['develop']="cmake"
-build_sys['mongodb-6.0']="cmake"
-build_sys['mongodb-5.0']="autoconf"
-build_sys['mongodb-4.4']="autoconf"
-build_sys['mongodb-4.2']="autoconf"
-build_sys['mongodb-4.0']="autoconf"
+#############################################################
+# Retrive the build system used by a particular release,
```

Comment:
Typo `Retrive`

Reply:
I didn't actually fix this. I'll come back to it